posthog/models/tagged_item.py

```diff
@@ -56,8 +73,10 @@ class TaggedItem(UUIDModel):
 
     class Meta:
         # Make sure to add new key to uniqueness constraint when extending tag functionality to new model
-        unique_together = ("tag",) + RELATED_OBJECTS
-        constraints = [models.CheckConstraint(check=build_check(), name="exactly_one_related_object",)]
+        constraints = [
+            *build_uniqueness_constraint(),
```

Comment:
non-blocking

Can we just add these constraints as, for example:

```
constraints = [
  build_partial_uniqueness_constraint(field="dashboard"),
  build_partial_uniqueness_constraint(field="insight"),
  ...
]
```

I find it quite indirect with `build_uniqueness_constraint` to see at a glance what constraints are added.

Reply:
One thing I wanted to avoid was to add one more place to check whenever someone wanted to add an object field to this model. I could also do something like: 

```
*[build_partial_uniqueness_constraint(field=o) for o in objects]
```

which might make it known what constraints we're actually making

Reply:
What does the addition of a new object field look like at the moment?